Assets/Editor/TestUnityEditorLoader.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Shopify.Tests {
+    using NUnit.Framework;
+    using UnityEngine.TestTools;
+    using UnityEngine;
+    using Shopify.Unity.SDK.Editor;
+    using Shopify.Unity.SDK;
+    using System.Collections;
+
+    [TestFixture]
+    public class TestUnityEditorLoader {
+        [UnityTest]
```

Comment:
This is a special testing feature that allows coroutines to be used so we can test the whole request cycle. This could be an integration test but the integration tests dont run in edit mode, and therefore dont give a good representation of the behaviour of the editor. These tests do run in edit mode so we can check the whole flow of the routine by making sure the request completes in the editor.